modules/delegates.js

```diff
@@ -17,6 +17,7 @@
 const crypto = require('crypto');
 const _ = require('lodash');
 const async = require('async');
+const lisk = require('lisk-elements').default;
```

Comment:
If we want to use `lisk-elements` we should add that to dependencies (we have only in dev dependencies).

Reply:
As mentioned in the PR already, the dependency change was already been part of another PR, so I didn't moved that commit here.